src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -147,6 +147,7 @@ def register_submission_attachment(
             "doc_vertrouwelijkheidaanduiding": "doc_vertrouwelijkheidaanduiding",
         },
     )
+    attachment_options["ontvangstdatum"] = submission.completed_on.date().isoformat()
```

Comment:
you'll want to make sure this is localize to Amsterdam time, see `openforms.utils.date.datetime_in_amsterdam`.

Otherwise a submission that was finished at 00:45 AM will be reported as submitted the day before :sparkles: because tiemzones are fun :sparkles: